purr/patterns/right-sidebar.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Title: Right Sidebar
+ * Slug: purr/right-sidebar
+ * Categories: sidebar
+ * Block Types: core/template-part/sidebar
+ * Description: Right Sidebar
```

Comment:
```suggestion
 * Description: Right Sidebar
 * Viewport width: 300
```

Reply:
This makes the preview look much better:

before | after
--- | ---
<img width="280" alt="Screenshot 2024-06-25 at 12 18 53" src="https://github.com/WordPress/community-themes/assets/3593343/157165e0-9870-4fae-89f6-7ccd32a34f13"> | <img width="283" alt="Screenshot 2024-06-25 at 12 19 45" src="https://github.com/WordPress/community-themes/assets/3593343/0c1e6cd5-d893-43bf-a976-331cf9977a7a">

Reply:
Added View port in Right Sidebar Pattern.